source/quickguides/quick-guide-generic.rst

```diff
@@ -0,0 +1,248 @@
+        param.test_name = '20161118.beta0.FC5COSP.ne30_ne30.edison'
+        param.multiprocessing = True
+        param.num_workers = 40
+        prefix = '#expand html_path#doc_examples'
+        param.results_dir = os.path.join(prefix, 'all_sets')
+        
+        #
+        ##Set specific parameters for new sets
+        ts_param = AreaMeanTimeSeriesParameter()
+        ts_param.reference_data_path = '#expand obs_path#time-series/'
+        ts_param.test_data_path = '#expand obs_path#time-series/E3SM_v1/'
+        ts_param.test_name = 'e3sm_v1'
+        ts_param.start_yr = '2002'
+        ts_param.end_yr = '2008'
+        
+        runner.sets_to_run = ['lat_lon','zonal_mean_xy', 'zonal_mean_2d', 'polar', 'cosp_histogram', 'meridional_mean_2d', 'area_mean_time_series']
+        runner.run_diags([param, ts_param])
+
+
+Advanced: Running custom diagnostics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The following steps are for 'advanced' users, who want to run custom diagnostics.
+So, most users will not run the software like this.
+
+
+By default, with ``e3sm_diags``,
+a built in set of variables are defined for each diagonostics sets.
+To do a short run, e.g. only running through a subset of variables,
+the configuration files is needed to customize the run.
+
+
+In the following example,
+only precipitation and surface sea temperature are run to compare with
+model and obs for lat_lon set.
+Create ``mydiags.cfg`` file as below.
+
+Check :doc:`Available Parameters <../available-parameters>` for all available parameters.
+
+For more examples of these types of files, look
+`here <https://github.com/E3SM-Project/e3sm_diags/blob/master/acme_diags/driver/default_diags/lat_lon_model_vs_obs.cfg>`_
+for the cfg file that was used to create all of the latitude-longitude sets.
+
+
+    ::
+
+        [#]
+        sets = ["lat_lon"]
+        case_id = "GPCP_v2.3"
+        variables = ["PRECT"]
+        ref_name = "GPCP_v2.3"
+        reference_name = "GPCP"
+        seasons = ["ANN", "DJF", "MAM", "JJA", "SON"]
+        regions = ["global"]
+        test_colormap = "WhiteBlueGreenYellowRed.rgb"
+        reference_colormap = "WhiteBlueGreenYellowRed.rgb"
+        diff_colormap = "BrBG"
+        contour_levels = [0.5, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14, 15, 16]
+        diff_levels = [-5, -4, -3, -2, -1, -0.5, 0.5, 1, 2, 3, 4, 5]
+        
+        
+        [#]
+        sets = ["lat_lon"]
+        case_id = "SST_HadISST"
+        variables = ["SST"]
+        ref_name = "HadISST"
+        reference_name = "HadISST/OI.v2"
+        seasons = ["ANN", "DJF", "MAM", "JJA", "SON"]
+        contour_levels = [-1, 0, 1, 3, 6, 9, 12, 15, 18, 20, 22, 24, 26, 28, 29]
+        diff_levels = [-5, -4, -3, -2, -1, -0.5, -0.2, 0.2, 0.5, 1, 2, 3, 4, 5]
+
+
+Run E3SM diagnostics with the ``-d`` parameter.
+Use the :ref:`run script linked here <cori-params-v2>`. And run as following:
```

Comment:
This link goes to `e3sm_diags/docs/html/quickguides/quick-guide-general.html#cori-params-v2`, which is actually `e3sm_diags/docs/html/quickguides/quick-guide-general.html#running-the-annual-mean-latitude-longitude-contour-set`.

What do we actually want to link to?

Reply:
source/quickguides/quick-guide-cori.rst has an anchor point `.. _cori-params-v2:` where it should refer to

Reply:
Same comment as above.

Reply:
I think it would be correct to link this to the `lat_lon` run script code block above?